modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -3870,6 +3882,23 @@ private void _validateGroupId(long groupId, String scope)
 		}
 	}
 
+	private void _validateListTypeEntryKey(
+			Map.Entry<String, Serializable> entry, String listTypeEntryKey,
+			ObjectField objectField)
+		throws PortalException {
+
+		ListTypeEntry listTypeEntry =
+			_listTypeEntryLocalService.fetchListTypeEntry(
+				objectField.getListTypeDefinitionId(), listTypeEntryKey);
+
+		if ((listTypeEntry == null) &&
+			(Validator.isNotNull(listTypeEntryKey) ||
+			 objectField.isRequired())) {
+
+			throw new ObjectEntryValuesException.ListTypeEntry(entry.getKey());
```

Comment:
@devpedrotavares I believe that `entry.getKey()` would be the the object field name, so we could use the `objectField.getName()` instead and simplify this method to receive only the `String listTypeEntryKey, ObjectField objectField`, makes sense?

Reply:
@carolmariaabb That's right! I just did this.